docs/_includes/examples/fakes/__tests.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+var proxyquire = require("proxyquire").noCallThru();
```

Comment:
I think there should be a comment here that says why we are using `proxyquire`. Immediately, it didn't make sense to me why we should be stubbing out `sinon` below, until it hit me that it's of course because we want to use the most recent version.